CHANGELOG.md

```diff
@@ -6,126 +6,132 @@
-- Removes `Publish-LocalUserExpiry` in favor of the `Publish-PasswordHealth` cmdlet. [GH-111](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/111)
+- **Breaking**: Updated `Invoke-VcfReportingPrereq` to:
+    - Use `-sddcManagerFqdn`, `sddcManagerUser`, and `sddcManagerPass` parameters to check the SDDC Manager version. [GH-117](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/117)
+    - Use `Write-LogMessage` to apply colors to the output and log the output to a file using the `-logPath` parameter. [GH-117](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/117)
+- **Breaking**: Updated `Invoke-VcfHealthReport` to use `-sddcManagerLocalUser` and `-sddcManagerLocalPass` parameters instead of `-SddcManagerRootPass`. Examples use the local `vcf` user account for the SDDC Manager virtual appliance. [GH-113](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/113)
+- **Breaking**: Updated `Publish-StorageCapacityHealth` to use `-localUser` and `-localPass` parameters instead of `-rootPass`. Examples use the local `vcf` user account for the SDDC Manager virtual appliance. [GH-113](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/113)
+- **Breaking**: Updated `Request-SddcManagerStorageHealth` to use `-localUser` and `-localPass` parameters instead of `-rootPass`. Examples use the local `vcf` user account for the SDDC Manager virtual appliance. [GH-113](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/113)
+- **Breaking**: Removed `Invoke-VcfPasswordPolicy` and supporting content. Please use the [`VMware.CloudFoundation.PasswordManagement`](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management) module. [GH-118](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/118)
+- Updated `Request-SoSHealthJson` to use the API to retrieve the SoS Health Summary JSON results. [GH-102](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/102)
+- Updated `Publish-PasswordHealth` to return the results from the SoS Health Summary JSON data. [GH-111](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/111)
+- Updated `Invoke-VcfHealthReport` to use the `Publish-PasswordHealth` cmdlet. [GH-111](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/111)
+- Removed `Publish-LocalUserExpiry` and supporting functions in favor of the `Publish-PasswordHealth` cmdlet. [GH-111](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/111)
+- Removed `Request-LocalUserExpiry` which supported the `Publish-LocalUserExpiry` cmdlet. [GH-120](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/120)
```

Comment:
Do you think we should add a little more context here? removed as SoS Health collects the required data or something ?